gen/dcos-config.yaml

```diff
@@ -1432,6 +1432,22 @@ package:
         # insecure_skip_verify = true
         [inputs.zookeeper.tags]
           dcos-component-name = "ZooKeeper"
+      # Read metrics from Admin Router Nginx Prometheus endpoint.
+      [[inputs.prometheus]]
+        ## An array of urls to scrape metrics from.
+        urls = ["https://127.0.0.1/nginx/metrics"]
+        ## Specify timeout duration for slower prometheus clients (default is 3s)
+        response_timeout = "10s"
+        ## Optional TLS Config
+        # tls_ca = /path/to/cafile
+        # tls_cert = /path/to/certfile
+        # tls_key = /path/to/keyfile
+        ## Use TLS but skip chain & host verification
+        insecure_skip_verify = true
+        ## Apply DC/OS component name tag according to the documentation.
+        [inputs.prometheus.tags]
+          dcos-component-name = "Admin Router"
```

Comment:
We could also call this Master Admin Router

Reply:
No, the name follows our Documentation:
https://docs.mesosphere.com/1.12/overview/architecture/components/
We could call it `Admin Router` and `Admin Router Agent` respectively though.

Reply:
Documentation is not always reference! :D